### PR TITLE
AC-929 Changed ProviderDashboardActivity to ProviderManagerDashboardActivity in Nav Graph

### PR DESCRIPTION
**Pull request by shivamsawlani02**
## Description of what I changed
Changed ProviderDashboardActivity to ProviderManagerDashboardActivity in Nav Graph.
Reference - Comments in this PR  https://github.com/openmrs/openmrs-contrib-android-client/pull/852

## Issue I worked on
JIRA Issue: https://issues.openmrs.org/browse/AC-929

## Checklist: I completed these to help reviewers :)
- [x] My pull request only contains **ONE single commit**

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by amtheshubham** [comment]
@shivamsawlani02 Also refactor the `directionToProviderManager` variable in `DashboardFragment.kt`

**Review by amtheshubham — COMMENTED** [review]
@shivamsawlani02 First change the following lines in the dashboard_nav_graph.xml
```
<action
            android:id="@+id/action_dashboardFragment_to_providerDashboardActivity"
            app:destination="@id/providerDashboardActivity" />
```
Then after this, rebuild the project. Then at last, change the variable of direction in the `DashboardFragment.kt`

Without all this, issue won't be solved

**Comment by shivamsawlani02** [comment]
> @shivamsawlani02 First change the following lines in the dashboard_nav_graph.xml
> 
> ```
> <action
>             android:id="@+id/action_dashboardFragment_to_providerDashboardActivity"
>             app:destination="@id/providerDashboardActivity" />
> ```
> 
> Then after this, rebuild the project. Then at last, change the variable of direction in the `DashboardFragment.kt`
> 
> Without all this, issue won't be solved

The issue is solved without this. But you are right. The names should be changed.

**Comment by amtheshubham** [comment]
> > @shivamsawlani02 First change the following lines in the dashboard_nav_graph.xml
> > ```
> > <action
> >             android:id="@+id/action_dashboardFragment_to_providerDashboardActivity"
> >             app:destination="@id/providerDashboardActivity" />
> > ```
> > 
> > 
> > Then after this, rebuild the project. Then at last, change the variable of direction in the `DashboardFragment.kt`
> > Without all this, issue won't be solved
> 
> The issue is solved without this. But you are right. The names should be changed.

It will create unnecessary confusion and hence will still remain an issue

**Comment by shivamsawlani02** [comment]
> > > @shivamsawlani02 First change the following lines in the dashboard_nav_graph.xml
> > > ```
> > > <action
> > >             android:id="@+id/action_dashboardFragment_to_providerDashboardActivity"
> > >             app:destination="@id/providerDashboardActivity" />
> > > ```
> > > 
> > > 
> > > Then after this, rebuild the project. Then at last, change the variable of direction in the `DashboardFragment.kt`
> > > Without all this, issue won't be solved
> > 
> > 
> > The issue is solved without this. But you are right. The names should be changed.
> 
> It will create unnecessary confusion and hence will still remain an issue

Yes I will change the names and then push.

**Comment by amtheshubham** [comment]
> > > > @shivamsawlani02 First change the following lines in the dashboard_nav_graph.xml
> > > > ```
> > > > <action
> > > >             android:id="@+id/action_dashboardFragment_to_providerDashboardActivity"
> > > >             app:destination="@id/providerDashboardActivity" />
> > > > ```
> > > > 
> > > > 
> > > > Then after this, rebuild the project. Then at last, change the variable of direction in the `DashboardFragment.kt`
> > > > Without all this, issue won't be solved
> > > 
> > > 
> > > The issue is solved without this. But you are right. The names should be changed.
> > 
> > 
> > It will create unnecessary confusion and hence will still remain an issue
> 
> Yes I will change the names and then push.

You can just rebuild the project, It will automatically be changed

**Comment by shivamsawlani02** [comment]
@amtheshubham have a look now.

**Comment by amtheshubham** [comment]
> @amtheshubham have a look now.

There are failed checks. Please rectify

**Comment by amtheshubham** [comment]
@shivamsawlani02 Rebase your local master branch with the origin(i.e.,OpenMRS master) so that your local master first receives the latest changes from the project. Then make a branch and commit.

**Comment by amtheshubham** [comment]
> I did not make much changes. I don't know why is Appveyor Build is failing. Travis passed successfully and the app is also working fine.

I will try rectifying this issue on my machine and then will update here

File: openmrs-client/src/main/res/navigation/dashboard_nav_graph.xml [hunk 1]
```diff
@@ -56,9 +56,9 @@
         android:name="org.openmrs.mobile.activities.activevisits.ActiveVisitsActivity"
         android:label="ActiveVisitsActivity" />
     <activity
-        android:id="@+id/providerDashboardActivity"
-        android:name="org.openmrs.mobile.activities.providerdashboard.ProviderDashboardActivity"
-        android:label="ProviderDashboardActivity" />
+        android:id="@+id/providerManagerDashboardActivity"
+        android:name="org.openmrs.mobile.activities.providermanagerdashboard.ProviderManagerDashboardActivity"
+        android:label="ProviderManagerDashboardActivity" />
```
Comment (amtheshubham) [inline openmrs-client/src/main/res/navigation/dashboard_nav_graph.xml:61]: @shivamsawlani02 add below two lines in place of line 61😄
```
android:label="activity_provider_management"
tools:layout="@layout/activity_provider_management" />
```

**Review by amtheshubham — CHANGES_REQUESTED** [review]

File: openmrs-client/src/main/res/navigation/dashboard_nav_graph.xml (hunk 1 above)
Comment (shivamsawlani02) [reply openmrs-client/src/main/res/navigation/dashboard_nav_graph.xml:61]: Oh! 

> @shivamsawlani02 add following two lines in place of line 61
> 
> ```
> android:label="activity_provider_management"
> tools:layout="@layout/activity_provider_management" />
> ```

Oh! It was missing in the other two activities also. Didn't notice.

Comment (amtheshubham) [reply openmrs-client/src/main/res/navigation/dashboard_nav_graph.xml:61]: No. It's there. It's just that there's a procedure to follow in Navigation Component. You just changed the variable names.

**Review by amtheshubham — COMMENTED** [review]
@shivamsawlani02 I only asked you to change line 61. Please don't change other lines and please don't underestimate this.

**Comment by shivamsawlani02** [comment]
> @shivamsawlani02 I only asked you to change line 61. Please don't change other lines and please don't underestimate this.

But then won't the other activities be different?

**Comment by shivamsawlani02** [comment]
> @shivamsawlani02 I only asked you to change line 61. Please don't change other lines and please don't underestimate this.

Ok. Won't change.

**Comment by amtheshubham** [comment]
@shivamsawlani02 The answer to your question is that the other two activities' names match with their layout files so no need of giving context to them. 
Moreover, It will compile but previous code is autogenerated by the system so I don't think you should mess with that code.

**Comment by shivamsawlani02** [comment]
> @shivamsawlani02 The answer to your question is that the other two activities' names match with their layout files so no need of giving context to them.
> Moreover, It will compile but previous code is autogenerated by the system so I don't think you should mess with that code.

@amtheshubham The app ran successfully on my mobile and now all the checks also passed and I think we should use the same naming convention in all the activities. If you wish I will change the code and remove context from the other two activities but since all checks passed I think it won't matter much and we can stick with this code.

**Comment by amtheshubham** [comment]
> > @shivamsawlani02 The answer to your question is that the other two activities' names match with their layout files so no need of giving context to them.
> > Moreover, It will compile but previous code is autogenerated by the system so I don't think you should mess with that code.
> 
> @amtheshubham The app ran successfully on my mobile and now all the checks also passed and I think we should use the same naming convention in all the activities. If you wish I will change the code and remove context from the other two activities but since all checks passed I think it won't matter much and we can stick with this code.

They are redundant lines and not needed. Please remove them

**Comment by amtheshubham** [comment]
LGTM!

**Comment by shivamsawlani02** [comment]
@rishabh-997 please review.

**Review by amtheshubham — APPROVED** [review]
LGTM!

**Pull request merged**